ember_debug/view-debug.js

```diff
@@ -614,10 +611,14 @@ var ViewDebug = Ember.Object.extend(PortMixin, {
    * @return {Boolean}
    */
   _isGlimmer: function() {
-    var id = Ember.keys(Ember.View.views)[0];
-    return id && !Ember.View.views[id].get('_childViews');
+    var id = Ember.keys(this.get('viewRegistry'))[0];
+    return id && !this.get('viewRegistry')[id].get('_childViews');
   },
 
+  viewRegistry: computed('application', function() {
+    return Ember.View.views || this.get('application.__container__').lookup('-view-registry:main');
```

Comment:
I reversed the order. You are right, I found the bug on 1.13 that removed `Ember.View.views`. PRing a fix to Ember core in a bit.